tfhe/src/c_api/high_level_api/mod.rs

```diff
@@ -22,23 +22,23 @@ mod zk;
 #[repr(C)]
 #[allow(non_camel_case_types)]
 pub enum FheTypes {
-    Type_FheBool,
+    Type_FheBool = 0,
     Type_FheUint2,
-    Type_FheUint4,
+    Type_FheUint4 = 1,
     Type_FheUint6,
-    Type_FheUint8,
+    Type_FheUint8 = 2,
     Type_FheUint10,
     Type_FheUint12,
     Type_FheUint14,
```

Comment:
We need all variants to have a value otherwise we'll have an issue later on, I'll check with BC for that and the fact we have more types than them

Reply:
we probably need a test (with a macro) to check that all values match between the C Enum and the HL one